sources/csv_reader/helpers.py

```diff
@@ -0,0 +1,16 @@
+def add_columns(columns, rows):
+    """Add column names to the given rows.
+
+    Args:
+        columns (List[str]): A list of column names.
+        rows (List[Tuple[Any]]): A list of rows.
+
+    Returns:
+        List[Dict[str, Any]]:
+            A list of rows with column names.
+    """
+    result = []
+    for row in rows:
+        result.append(dict(zip(columns, row)))
+
+    return result
```

Comment:
`duckdb` returns data from CSV in a form of tuples, without column names. It, of course, causes a very unpretty scheme of destination tables. I thought, it can be solved by turning tuples into dicts.